Barotrauma/BarotraumaShared/SharedSource/Items/Components/ItemContainer.cs

```diff
@@ -492,6 +491,12 @@ public override void Move(Vector2 amount, bool ignoreContacts = false)
 
         public void OnItemRemoved(Item containedItem)
         {
+            foreach (ActiveContainedItem activeContainedItem in activeContainedItems)
+            {
+                if (activeContainedItem.Item != containedItem || !TryProcessActiveContainedItem(activeContainedItem)) continue;
+                activeContainedItem.StatusEffect.Apply(ActionType.OnRemoved, 1, item, targets);
```

Comment:
Named arguments should be used if it's not obvious from the context what an argument means. Here it's not clear what the "1" means without looking at the method's signature.